libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -900,18 +900,6 @@ public String getAccountType() {
         return context.getString(getSalesforceR().stringAccountType());
     }
 
-    /**
-     * Indicates whether the app is running on a tablet.
-     *
-     * @return True if the application is running on a tablet.
-     */
-    public static boolean isTablet() {
```

Comment:
This is an inaccurate API that has outlived it's purpose. We can't make the determination of phone (vs) tablet, with a 7" screen in the mix. Also, with the advent of fragments, this API means very little now. It's not being used anywhere, since we switched to ```ActionBar```.